src/triggers.js

```diff
@@ -588,6 +639,142 @@ export function maybeRunQueryTrigger(
     );
 }
 
+export function resolveError(message, defaultOpts) {
+  if (!defaultOpts) {
+    defaultOpts = {};
+  }
+  if (message instanceof Parse.Error) {
+    if (!message.message && defaultOpts.message) {
+      message.message = defaultOpts.message;
```

Comment:
Add a test case for this. message.message is undefined if you use `Parse.Error(code)` without message.

Reply:
Sharp eyes!

Reply:
Code Coverage isn't at 100%. Check it out